api/nakadi-event-bus-api.yaml

```diff
@@ -1246,7 +1254,16 @@ definitions:
     properties:
       name:
         type: string
-        description: Name of the strategy.
+        enum: ['hash', 'user_defined', 'random']
```

Comment:
Users can ask the api to get a list of available strategies and their documentation.